zokrates_stdlib/stdlib/ecc/babyjubjubParams.zok

```diff
@@ -14,31 +14,17 @@ struct BabyJubJubParams {
 	field Gv
 }
 
-def main() -> BabyJubJubParams:
-
-	// Order of the curve for reference: 21888242871839275222246405745257275088614511777268538073601725287587578984328
-    field JUBJUBC = 8   // Cofactor
-    field JUBJUBA = 168700    // Coefficient A
-    field JUBJUBD = 168696    // Coefficient D
-    field MONTA = 168698    // int(2*(JUBJUB_A+JUBJUB_D)/(JUBJUB_A-JUBJUB_D))
-    field MONTB = 1    // int(4/(JUBJUB_A-JUBJUB_D))
-
-// Point at infinity
-    field[2] INFINITY = [0, 1]
-
-//  Generator
-    field Gu = 16540640123574156134436876038791482806971768689494387082833631921987005038935
-    field Gv = 20819045374670962167435360035096875258406992893633759881276124905556507972311
-
-// Index
-//         0       1         2            3         4   5      6       7        8      10
-return BabyJubJubParams {
-	JUBJUBA: JUBJUBA,
-	JUBJUBD: JUBJUBD,
-	INFINITY: INFINITY,
-	Gu: Gu,
-	Gv: Gv,
-	JUBJUBC: JUBJUBC,
-	MONTA: MONTA,
-	MONTB: MONTB
+// Order of the curve for reference: 21888242871839275222246405745257275088614511777268538073601725287587578984328
+const BabyJubJubParams BABYJUBJUB_PARAMS = BabyJubJubParams {
+    JUBJUBC: 8,       // Cofactor
+    JUBJUBA: 168700,  // Coefficient A
+    JUBJUBD: 168696,  // Coefficient D
+    MONTA: 168698,    // int(2*(JUBJUB_A+JUBJUB_D)/(JUBJUB_A-JUBJUB_D))
+    MONTB: 1,         // int(4/(JUBJUB_A-JUBJUB_D))
+    INFINITY: [0, 1], // Point at infinity
+    Gu: 16540640123574156134436876038791482806971768689494387082833631921987005038935, // Generator
+    Gv: 20819045374670962167435360035096875258406992893633759881276124905556507972311 // ...
 }
+
+def main() -> BabyJubJubParams:
```

Comment:
We could omit that and just use the constant?

Reply:
I had second thoughts about this, so I left it in so you could still use the `import` statement on this file. A bit of a breaking change here, but should be fine as we will remove the `import` statement eventually.